### PR TITLE
javascript-timers | Grace Park

### PR DESCRIPTION
**Pull request by igracepark**
https://user-images.githubusercontent.com/44829206/151450583-683de928-c15c-4bc5-9b73-84230f01e40a.mov

https://user-images.githubusercontent.com/44829206/151450593-21b74f3c-b08e-4d81-9784-c3092482af6a.mov

Screen Shot 2022-01-27 at 1 58 48 PM
Screen Shot 2022-01-27 at 1 58 42 PM

### CODE REVIEW

File: javascript-timers/timeout.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var headingElement = document.querySelector('h1');
+
+function helloThere() {
+  headingElement.innerHTML = 'Hello There';
```
Comment (TimelikeClosure) [inline javascript-timers/timeout.js:4]: It's generally a bad idea to set `innerHTML` directly (for both performance and security reasons). Please find another way to modify the text.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-timers/timeout.js (hunk 1 above)
Comment (igracepark) [reply javascript-timers/timeout.js]: Screen Shot 2022-01-27 at 7 49 19 PM

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var headingElement = document.querySelector('h1');
+var counter = 5;
+
+function countDown() {
+  counter--;
+  headingElement.innerHTML = counter;
+  if (counter === 0) {
+    clearInterval(interval);
+    headingElement.innerHTML = '~Earth Beeeelooowww Us~';
```
Comment (TimelikeClosure) [inline javascript-timers/interval.js:9]: Please don't set `innerHTML` here either

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-timers/interval.js (hunk 2 above)
Comment (igracepark) [reply javascript-timers/interval.js]: Screen Shot 2022-01-28 at 1 02 17 PM
Screen Shot 2022-01-28 at 1 02 10 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**